test/functional/wcow_test.go

```diff
@@ -172,50 +172,41 @@ import (
 //}
 
 // Helper to start a container.
-// Ones created through hcsoci methods will be of type *hcs.System.
+// Ones created through hcsoci methods will be of type cow.Container.
 // Ones created through hcsshim methods will be of type hcsshim.Container
-func startContainer(t *testing.T, c interface{}) {
-	var err error
-	switch c.(type) {
-	case *hcs.System:
-		err = c.(*hcs.System).Start()
-	case hcsshim.Container:
-		err = c.(hcsshim.Container).Start()
-	default:
-		t.Fatal("unknown type")
-	}
+func startContainer(t *testing.T, c interface{ Start() error }) {
```

Comment:
`CreateContainer` now returns `cow.Container` can we just change `c` to `c cow.Container`

Reply:
It's also called with an hcsshim.Container.

Reply:
Ah. nvm then